### DISCUSSION TITLE
I want say Thank you

### DISCUSSION THREAD

**Discussion by dalisoft** [ravynsoft/ravynos#378] [General]
Hi @ravynsoft

I did like the idea of open source ravynos. I am made dream about OS with gaming performance (like DLSS/FSR) of Windows, Apps & quality of macOS and performance (hardware) of Linux. That would be just super OS.

Currently all that i can do is just testing (started downloading now). Then i may join to development with ideas

**Comment by mszoek** [comment]
It's not very usable yet, but I appreciate the kind words :)
